### ISSUE TITLE
[FEATURE_REQUEST] Guest Access 

### ISSUE THREAD

**Issue by MuadDibVV** [Lissy93/dashy#56]
Is it possible to have a guest access without any credential and an admin access with a button for an admin use ?

**Comment by Lissy93** [comment]
Hmm, yes it could be possible. So that no login is required for view-only access, but the user must be authenticated as an admin in order for config changes to be written to the disk from the UI?

**Comment by MuadDibVV** [comment]
yes
I thought that allowConfigEdit: false would meet my need (configuration only by yaml (I use the docker version)) but because I don't use users it doesn't seem to work

**Comment by Lissy93** [comment]
Oh that (`allowConfigEdit: false`) actually should work. A user can still apply changes locally, but those changes will only ever be saved in their browsers local storage. So if `allowConfigEdit: false` is set, then no user can write changes to the `conf.yml` file through the UI.

**Comment by MuadDibVV** [comment]
Ok thanks, so I confirm that a guest access could be interesting

**Comment by Lissy93** [comment]
Sorry that this took me so long to get round to. I initially thought that users who'd want more advanced login controls would be hosting their own authentication server, but I think I was wrong.

So I've implemented this in PR #167 which I will merge now, and will be released in V 1.6.2. You will then be able to use it by setting `appConfig.enableGuestAccess: true`

<details>
	<summary>Example</summary>
	<p>

```yaml
appConfig:
  enableGuestAccess: true # Now login isn't required to access dashboard
  auth: # You still need to have users though
    - user: alicia
      hash: 4D1E58C90B3B94BCAD9848ECCACD6D2A8C9FBC5CA913304BBA5CDEAB36FEEFA3
      type: admin
    - user: bob
      hash: 5E884898DA28047151D0E56F8DC6292773603D0D6AABBDD62A11EF721D1542D8
```
</p>
</details>

Then, even when not logged in you will be able to view your dashboard. Guests cannot write any changes to disk, but can modify settings locally (saved just in their browser). Click the login button in the top-right to sign in as your normal user in order to make changes to your config file.

**Comment by Lissy93** [comment]
@MuadDibVV - Is this still something you want included? I am unsure weather to keep it in the app, as it adds a lot of logic + code for a small feature? Do let me know so I don't revert something you plan to use.
